.travis.yml

```diff
@@ -4,3 +4,9 @@ language: java
 
 jdk:
   - oraclejdk8
+
+install:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -PTravis
```

Comment:
Travis executes `mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V` without profile.

Reply:
But my question is why do we need install, then script step.

Reply:
Original plan was to include something like `mvn test -s settings.xml -PTravis` to execute OracleDB tests. Since we dont execute tests depending on Orable jars on Travis it can be removed

Reply:
Observed Travis build fails with `mvn test -B`. We need to include script test also